### DISCUSSION TITLE
Support for Affinity file types

### DISCUSSION THREAD

**Discussion by gitarian** [ccseer/Seer#389] [General]
Hi,

 I would like to ask for support for (major) file types provided by the [Affinity Graphic Suite](https://affinity.serif.com/).
 
Though there are different extensions on the file system (like .afphoto, .afdesign, .afbook, .afpub etc.), they all share the same file format/content definition, afaik.
 
Let me know if I can help further.

Thanks,
gitarian (Seer Pro user)

**Comment by gitarian** [comment]
Hmm, maybe I should have better posted my request in the Discussions section?

**Comment by gitarian** [comment]
Thanks for moving - and sorry for my mistake. ;-)

Multi edit: fixed typos in my previous posts.

**Comment by ccseer** [comment]
Affinity files (.afphoto, .afdesign, .afpub, .af) use a proprietary format with no published specification, so third-party support is extremely limited — even other image viewers like XnViewMP can only show a low-resolution embedded preview rather than the full file.

It's possible to extract embedded preview for display in Seer, which would at least let you see what the file contains at a glance. Full rendering isn't feasible without official format documentation from Serif/Canva.

I'll add your vote to the tracker. 
Thanks for taking the time to report this!
